config/gulp/scripts.js

```diff
@@ -27,7 +27,9 @@ function compileCommon() {
 }
 
 function compile() {
-  return src([`${PROJECT_JS_SRC}/*.js`])
+  // TODO: Move USWDS init to a vendor directory.
+  // That way we can ignore vendor instead of individual files.
+  return src([`${PROJECT_JS_SRC}/*.js`, `!${PROJECT_JS_SRC}/uswds-init.min.js`])
```

Comment:
`uswds-init.min.js` is a vendor file. Ignoring it in jshint because it shows 10 errors on `npm start` command.


**The errors**
```bash
10 errors
themes/digital.gov/src/js/uswds-init.min.js: line 1, col 195, Missing semicolon.
themes/digital.gov/src/js/uswds-init.min.js: line 1, col 195, Expected an identifier and instead saw '='.
themes/digital.gov/src/js/uswds-init.min.js: line 1, col 195, Expected an assignment or function call and instead saw an expression.
themes/digital.gov/src/js/uswds-init.min.js: line 1, col 196, Missing semicolon.
themes/digital.gov/src/js/uswds-init.min.js: line 1, col 215, Expected an assignment or function call and instead saw an expression.
themes/digital.gov/src/js/uswds-init.min.js: line 1, col 216, Missing semicolon.
themes/digital.gov/src/js/uswds-init.min.js: line 1, col 295, Missing semicolon.
themes/digital.gov/src/js/uswds-init.min.js: line 1, col 316, Expected an assignment or function call and instead saw an expression.
themes/digital.gov/src/js/uswds-init.min.js: line 1, col 317, Missing semicolon.
themes/digital.gov/src/js/uswds-init.min.js: line 1, col 217, 't' used out of scope.
themes/digital.gov/src/js/uswds-init.min.js: line 1, col 250, 't' used out of scope.
themes/digital.gov/src/js/uswds-init.min.js: line 1, col 297, 't' used out of scope.
themes/digital.gov/src/js/uswds-init.min.js: line 1, col 299, 't' used out of scope.
themes/digital.gov/src/js/uswds-init.min.js: line 1, col 337, Missing semicolon.
themes/digital.gov/src/js/uswds-init.min.js: line 1, col 419, Missing semicolon.
themes/digital.gov/src/js/uswds-init.min.js: line 1, col 565, Missing semicolon.
themes/digital.gov/src/js/uswds-init.min.js: line 1, col 800, Expected an assignment or function call and instead saw an expression.
themes/digital.gov/src/js/uswds-init.min.js: line 1, col 801, Missing semicolon.
themes/digital.gov/src/js/uswds-init.min.js: line 1, col 807, Missing semicolon.
themes/digital.gov/src/js/uswds-init.min.js: line 1, col 810, Expected an assignment or function call and instead saw an expression.
themes/digital.gov/src/js/uswds-init.min.js: line 1, col 811, Missing semicolon.
themes/digital.gov/src/js/uswds-init.min.js: line 1, col 824, Expected an assignment or function call and instead saw an expression.
```